packages/nvidia_nat_harbor/src/nat_harbor/agents/installed/nemo_agent.py

```
@@ -0,0 +1,284 @@
+            default="python3",
+            env_fallback="NVIDIA_NAT_PYTHON_BIN",
+        ),
+        # Enables inline (in-process) NAT execution instead of shell wrapper.
+        CliFlag(
+            "library_mode",
+            cli="--library-mode",
+            type="bool",
+            default=False,
+            env_fallback="NAT_HARBOR_LIBRARY_MODE",
+        ),
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)